index.html

```diff
@@ -1489,90 +1489,60 @@ <h2>Cryptographic Suites</h2>
-representation and always transforms it into a deterministic representation. For
-example, alphabetically sorting a list of items is a type canonicalization. This
-process is sometimes also called <em>normalization</em>.
-        </dd>
-        <dt><dfn>message digest algorithm</dfn></dt>
-        <dd>
-An algorithm that takes an input message and produces a cryptographic
-output message that is often many orders of magnitude smaller than the
-input message. These algorithms are often 1) very fast, 2)
-non-reversible, 3) cause the output to change significantly when even one
-bit of the input message changes, and 4) make it infeasible to find two
-different inputs for the same output.
-        </dd>
-        <dt><dfn>proof algorithm</dfn></dt>
-        <dd>
-An algorithm that takes an input message and produces an output value where the
-receiver of the message can mathematically verify that the message has not
-been modified in transit and came from someone possessing a particular secret.
-        </dd>
-      </dl>
-
-      <p>
-At a minimum, a <a>proof type</a> is expected have the following attributes:
-      </p>
-
-      <dl style="margin-left: 1em;">
-        <dt>id</dt>
-        <dd>
-A URL that identifies the <a>cryptographic suite</a>.
-For example: <code>https://w3id.org/security#Ed25519Signature2020</code>.
-        </dd>
-        <dt>type</dt>
-        <dd>
-The value <code>ProofSuite</code>.
-        </dd>
-        <dt>canonicalizationAlgorithm</dt>
-        <dd>
-A URL that identifies the <a>canonicalization algorithm</a> to use on the
-<var>document</var>. For example:
-<code>https://w3id.org/security#URDNA2015</code>.
-        </dd>
-        <dt>digestAlgorithm</dt>
-        <dd>
-A URL that identifies the <a>message digest algorithm</a> to use on the
-<var>canonicalized document</var>. For example:
-<code>https://www.ietf.org/assignments/jwa-parameters#SHA256</code>
-        </dd>
-        <dt>proofAlgorithm</dt>
-        <dd>
-A URL that identifies the <a>proof algorithm</a> to use on the
-<var>data to be signed</var>. For example:
-<code>https://w3id.org/security#ed25519</code>
-        </dd>
-      </dl>
+      <ul>
+        <li>
+The specification MUST be published as a human-readable document at a URL.
+        </li>
+        <li>
+The specification MUST identify a cryptographic suite <a>type</a> and any
+parameters that can be used with the suite.
+        </li>
+        <li>
+The specification MUST detail the algorithms used to perform transformations,
+if any, to the data to be protected.
+        </li>
+        <li>
+The specification MUST detail the algorithms used to perform cryptographic
+hashing to the data to be protected.
+        </li>
+        <li>
+The specification MUST detail the algorithms used to perform cryptographic
+protection of the data.
+        </li>
+        <li>
+The specification MUST detail the algorithms used to perform cryptographic
+verification of the data.
```

Comment:
```suggestion
The specification MUST detail the algorithms, parameters, and other necessary details used to perform cryptographic verification of the data.
```

Reply:
I have no particular argument with this suggestion, but I submit that if this change is made, it should also be echoed in all the nearby list items.